### PR TITLE
refactor data pipeline

### PR DESCRIPTION
**Pull request by ghost**
This PR refactors and cleans up the data pipeline, no functionality changed

### CODE REVIEW

File: model/utils/data_feed.py [hunk 1]
```diff
@@ -7,38 +7,24 @@
...
-print(data_folder.resolve())
-data_feed = DataFeed(data_folder=data_folder.resolve())
-try:
-    data_feed.load_historical_data('mock_logreturns.prq')
-except FileNotFoundError as e:
-    raise RuntimeError(
-        "Run `python data/mock_data.py` to generate the mock_logreturns.prg file"
-        ) from e
+data_folder = Path(__file__, "../../../data/").resolve()
+data_file = "mock_logreturns.prq"
```
Comment (bowd) [inline model/utils/data_feed.py:30]: Move these two to the top of the file and make them constants
```
DATA_FOLDER = Path(__file__, "../../../data/").resolve()
DATA_FILE = "mock_logreturns.prq"
```

File: model/utils/data_feed.py [hunk 2]
```diff
@@ -7,38 +7,24 @@
...
-
-        if file_name[-3:] == "csv":
-            historical_data = pd.read_csv(Path(self.data_folder, file_name))
-        elif file_name[-3:] == "prq":
-            historical_data = pd.read_parquet(Path(self.data_folder, file_name))
+        reads historical log returns
+        """
+        historical_data = pd.read_parquet(Path(self.data_folder, data_file))
         self.data = np.array(historical_data)
         self.length = len(historical_data)
```
Comment (bowd) [inline model/utils/data_feed.py:26]: Make this part of the initializer and remove this function, it doesn't really make sense at this point.

**Review by bowd — CHANGES_REQUESTED** [review]

File: model/utils/data_feed.py (hunk 1 above)
Comment (sissnad) [reply model/utils/data_feed.py]: Just thinking. With mock_data it does not matter of course. But in general you want the data file to be flexible with both name and format. At least an csv option. Most users will probably want to try it on csv. I understand that it won't work in `system_parameters.py`. But how about `simulation.configuration.py`?

File: model/utils/data_feed.py (hunk 2 above)
Comment (sissnad) [reply model/utils/data_feed.py]: See below, I have the feeling offering some format flexibility would be nice, no?

**Comment by tobikuhlmann** [comment]
@sissnad yeah true ... I was thinking this will change and we refactor anyways when we start to use real historical data and document how others can, so I thought I make it as simple as possible now

**Comment by sissnad** [comment]
> @sissnad yeah true ... I was thinking this will change and we refactor anyways when we start to use real historical data and document how others can, so I thought I make it as simple as possible now

Let's open an issue to not forget and have it as a low priority in the epic. What you think?

**Comment by tobikuhlmann** [comment]
yes thats good! https://app.zenhub.com/workspaces/mento--reserve-board-61b07ac76f46fe00100b2a16/issues/celo-org/mento2-model/125

**Review by sissnad — APPROVED** [review]

**Review by bowd — APPROVED** [review]

**Pull request merged**